antismash/detection/genefinding/__init__.py

```diff
@@ -49,7 +49,9 @@ def check_prereqs() -> List[str]:
     if options.genefinding_tool in ['none']:
         return failure_messages
     binaries = []  # type: List[str]
-    if options.genefinding_tool in ['prodigal', 'prodigal-m']:
+    if options.check_prereqs_only:
+        binaries = ["prodigal", "glimmerhmm"]
```

Comment:
hmm, not sure about this one. if you never run on fungi, you never need glimmerh,,

Reply:
otoh, you can just add a dummy script in that case